src/ipyaladin/widget.py

```diff
@@ -389,6 +394,58 @@ def target(self, target: Union[str, SkyCoord]) -> None:
             }
         )
 
+    def add_hips(self, hips: str, **hips_options: any) -> None:
```

Comment:
I would suggest to rename hips_options -> options because we already know it concerns the add of an hips.